test/bigtest/interactors/job-log-tab.js

```diff
@@ -0,0 +1,15 @@
+import {
+  interactor,
+  scoped,
+  text,
+} from '@bigtest/interactor';
+
+@interactor
+class TabViewInteractor {
+  total = scoped('#view-total-records-test');
+  textTotal = text('#view-total-records-test');
```

Comment:
`textTotal` and `textJson ` seems like excess here. Try to use existing `total` and `logJson` and their text properties `total.text`.